README.md

```diff
@@ -1,5 +1,8 @@
 # Daggy
 
+[![Build Status](https://img.shields.io/travis/fantasyland/daggy.svg?branch=master)](https://travis-ci.org/fantasyland/daggy)
```

Comment:
I believe the correct format is `https://img.shields.io/travis/USER/REPO/BRANCH.svg`.